src/scenic/syntax/ast.py

```diff
@@ -5,6 +5,8 @@
 class AST(ast.AST):
     "Scenic AST base class"
 
+    _attributes = ("lineno", "col_offset", "end_lineno", "end_col_offset")
```

Comment:
`_attributes` were needed for `copy_location` and other utilities functions from the `ast` module to properly function.

https://github.com/python/cpython/blob/f6fd8aac13714ce17650eb4a648d5c08f0be53b4/Lib/ast.py#L178-L192

https://github.com/python/cpython/blob/f6fd8aac13714ce17650eb4a648d5c08f0be53b4/Lib/ast.py#L195-L227